app/templates/events/event_list.html

```diff
@@ -65,7 +65,11 @@ <h1 class="text-center mt-5">Events List for {{selectedTerm.description}}</h1>
     <tbody>
         {% for event in events %}
           <tr>
-            <td> <a href="/eventsList/{{event.id}}/edit" class="link-primary">{{event.name}}</a></td>
+              {% if user.isCeltsAdmin or user.isCeltsStudentStaff %}
```

Comment:
student staff can only edit if they are the program manager for the program the event is in